services/mqtt/config.go

```diff
@@ -0,0 +1,53 @@
+package mqtt
+
+import (
+	"errors"
+	"net"
+	"net/url"
+	"strconv"
+)
+
+type Config struct {
+	// Enabled indicates whether the service should be enabled
+	Enabled bool `toml:"enabled" override:"enabled"`
+	// URL of the MQTT Broker
+	Host string `toml:"host" override:"host"`
+	// Port of the MQTT Broker
+	Port uint16 `toml:"port" override:"port"`
+
+	ClientID string `toml:"client-id" override:"client-id"`
+	Username string `toml:"username" override:"username"`
+	Password string `toml:"password" override:"password"`
```

Comment:
If you update this `override:"password"` to `override:"password,redact"` then the config API will redact the password in all relevant API calls.